app.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "Heroku: ns-notifier",
+    "description": "A Flask App that send sms alerts about glucose level to your family or friends.",
+    "image": "heroku/python",
+    "repository": "https://gitlab.com/codeonrocks/client/nexmo-nsnotifier",
```

Comment:
This points to a private fork of this repository

Reply:
@superdiana thanks for the observation, you are totally right. I changed this information in the last commit.